packages/api/src/config.js

```diff
@@ -25,6 +25,7 @@ export const products = {
       },
     ],
     monthlyPrice: 0,
+    viewerLimit: 30,
```

Comment:
What are these config changes?

Reply:
It's the viewer limit specified per product. In the future (billing v2) it will be a column in the `product` table, with a default viewerLimit specified on a specific product